thoth/storages/ceph.py

```diff
@@ -97,6 +99,10 @@ def retrieve_blob(self, object_key: str) -> bytes:
                 raise NotFoundError("Failed to retrieve object, object {!r} does not exist".format(object_key)) from exc
             raise
 
+    def retrieve_document_last_modification_date(self, object_key: str) -> datetime:
+        """Retrieve the last modification date of a document from S3."""
+        return self._s3.Object(self.bucket, f"{self.prefix}{object_key}").get()["LastModified"]
```

Comment:
I feel like we should probably not have a method for something so specific.

What about 
```suggestion
    def retrieve_document_attr(self, object_key: str, attr: str) -> datetime:
        """Retrieve the given attribute of a document from S3."""
        return self._s3.Object(self.bucket, f"{self.prefix}{object_key}").get()[str]
```
and calling that with "LastModified" ?

Maybe I'm worried over nothing. I'm not sure of the correct ratio specialize/generalize.

Reply:
(leaving the method out completely like I initially suggested does not seem practical)